firebase-admin/runtime/pom.xml

```diff
@@ -14,6 +14,7 @@
 
     <properties>
         <firebase-admin-sdk.version>9.1.1</firebase-admin-sdk.version>
+        <microprofile-jwt-auth-api.version>1.2.1</microprofile-jwt-auth-api.version>
```

Comment:
Is it really needed ? Isn't it managed by the Quarkus BOM ?

Reply:
I don't think microprofile jwt is managed by BOM since I'm getting build errors.